edx_notifications/management/commands/tests/test_background_check.py

```diff
@@ -67,3 +67,16 @@ def test_timer_execution(self):
 
         self.assertIsNotNone(readback_timer.executed_at)
         self.assertIsNone(readback_timer.err_msg)
+
+
+class PurgeNotificationsCommandTest(TestCase):
+    """
+    Test suite for the management command
+    """
+
+    def test_purge_command_check(self):
+        """
+        Invoke the Management Command
+        """
+
+        force_purge.Command().handle()
```

Comment:
hate to say this, but this isn't a real test. It certainly will get us some code coverage stats. :-) 

Can you set up some test data and make sure it got deleted? Thanks.